src/hooks/useENSRegistrationForm.ts

```diff
@@ -195,20 +199,30 @@ export default function useENSRegistrationForm({
 
   const onChangeField = useCallback(
     ({ key, value }) => {
-      if (!isEmpty(errors)) {
-        setErrors(errors => {
-          const newErrors = omit(errors, key);
-          return newErrors;
-        });
+      setEvaluatingErrors(true);
+      const validation = textRecordFields[key as ENS_RECORDS]?.validation;
+      if (validation) {
+        const { message, validator } = validation;
+        const isValid = !value || validator(value);
```

Comment:
we may need to debounce this... might not be that performant if we are applying validations on every keystroke (and might be a weird UX to see flashing errors as you're typing).

Reply:
This method, `onChangeField`, is only used in one place (`TextRecordsForm`) and is already debounced by 300ms there. Will that suffice?